packages/replica-healthcheck/package.json

```diff
@@ -22,6 +22,7 @@
   "devDependencies": {
```

Comment:
I don't think that all of these deps are `devDependencies`

Reply:
which one's not?? `dotenv` maybe since it's required to run?

Reply:
^switched that one!